samples/caf/src/main.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <event_manager.h>
```

Comment:
No need to introduce event_manager header directly

Reply:
Its a copy paste `main.c` from our applications. This is what we suppose to include as we are calling `event_manager_init()`. Its described in the documentation. I wouldn't change that to avoid confusion.

Reply:
I missed that one, we actually use event_manager_init so header should be explicitly included here. Sorry for the confusion.